### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by masterashu** [masterashu/TestCaseGenerator#3]
Add list of dependencies i.e. python 3.6 +
Fix the run program command. 
Add link to examples provided in [samples.md](https://github.com/masterashu/TestCaseGenerator/blob/master/samples.md)

**Comment by dwijenchawra** [comment]
I'd like to work on this PR! I'm new to open-source, how do I make this count to Hacktoberfest?

**Comment by masterashu** [comment]
For making a PR Count in Hacktober Fest, you should make a vaild( should not be labelled as spam) PR in any issue labelled as `hactoberfest` in the month of october.

**Comment by dwijenchawra** [comment]
Ok, sounds good! I can work on it now, and then make the pull request in October, right?

**Comment by masterashu** [comment]
Yes and make sure you also make a commit in October as well before making a pull request.

**Comment by navyakanu** [comment]
#17 
Sorry, raised a PR without looking at the comments, not sure if this was already taken!
